src/core/constants.js

```diff
@@ -1,7 +1,7 @@
 /** @module */
 
 /** The current lib version, reported with errors and analytics */
-export const LIB_VERSION = 'v1.3.1';
+export const LIB_VERSION = '@@LIB_VERSION';
```

Comment:
Let's add a comment(s) indicating that this value is now injected. That could help minimize confusion for any new person navigating our code.

Reply:
done in the comment above the const